libraries/rush-lib/src/schemas/pnpm-lock-validation.schema.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
Please update the `rush init` template to include your new setting:

https://github.com/microsoft/rushstack/blob/main/libraries/rush-lib/assets/rush-init/common/config/rush/pnpm-config.json

I can help with docs